torch/_inductor/config.py

```diff
@@ -180,3 +180,15 @@ class trace:
     # Upload the .tar.gz file
     # Needs to be overriden based on specific environment needs
     upload_tar = None
+
+
+class TritonConfigContext:
```

Comment:
Suggest renaming this to InductorConfigContext.  Cudagraphs is not triton-specific.